### PR TITLE
feat: hide breadcrumbs when loading submission list

### PR DESCRIPTION
**Pull request by muselesscreator**
noted missing workflow from previous ticket.
image

### CODE REVIEW

File: src/containers/ListView/index.jsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ export class ListView extends React.Component {
     const { isLoaded, hasError } = this.props;
     return (
       <Container className="py-4">
-        <ListViewBreadcrumb />
+        { isLoaded && <ListViewBreadcrumb /> }
```
Comment (leangseu-edx) [inline src/containers/ListView/index.jsx:35]: Just a nit
```
{
  isLoaded && <><ListVie... </>
}
```

Comment (muselesscreator) [reply src/containers/ListView/index.jsx:35]: I thought of that too, but thought this looked a little bit cleaner (2 vs 6 lines)

Comment (muselesscreator) [reply src/containers/ListView/index.jsx:35]: and I didn't like the inline-implication of:
`{ isLoaded && (<><ListViewBreadcrumb /><SubmissionTable /></>)}`

Comment (leangseu-edx) [reply src/containers/ListView/index.jsx:35]: Not sure how I feel 😅

Comment (leangseu-edx) [reply src/containers/ListView/index.jsx:35]: We can roll with this.

**Review by leangseu-edx — APPROVED** [review]
LGTM

**Pull request merged**